data-prepper-plugins/opensearch/README.md

```diff
@@ -129,6 +129,8 @@ If not provided, failed records will be written into the default data-prepper lo
 all the records received from the upstream prepper at a time will be sent as a single bulk request.
 If a single record turns out to be larger than the set bulk size, it will be sent as a bulk request of a single document.
 
+- `ism_policy_file` (optional): A String of absolute file path to an ISM (Index State Management) policy file in JSON format. This policy file is effective only when there is no built-in policy file for the index type. For example, `custom` index type is currently the only one without a built-in policy file, thus it would use the policy file here if it's provided through this parameter. OpenSearch documentation has more about [ISM policies.](https://opensearch.org/docs/latest/im-plugin/ism/policies/)
```

Comment:
It might be clearer if it is reworded as:

> A String of absolute file path for an ISM(Index State Management) policy JSON file

Reply:
How about just?

> The absolute file path for an ISM (Index State Management) policy JSON file. ...

Reply:
I thought we wanted to be explicit with the data type but it's obvious a file path will be String. ^ LGTM
Might be worth rewording `dlq_file` in a similar manner.

Reply:
Here I lean toward keeping the style consistent across parameter descriptions, so I am using "A String of absolute file path for an ISM(Index State Management) policy JSON file" like what Shivani suggested. We can have more discussion if you feel strong about this.

Reply:
Maybe in the future, we can follow the pattern here: https://opensearch.org/docs/latest/monitoring-plugins/trace/data-prepper-reference/

e.g.
> String, path to a .jks or .p12 keystore file. Required if ssl is true.

Reply:
Yeah, that would be nicer.